test/test_performance.py.in

```diff
@@ -10,6 +10,8 @@ import unittest
 import launch
 from launch import LaunchDescription
 from launch_ros.actions import Node
+from launch.event_handlers import OnProcessStart
+from ament_index_python import get_package_prefix
```

Comment:
These can be dropped.

Reply:
https://github.com/ros2/buildfarm_perf_tests/pull/26/commits/ffe6e7bdbefceea74a77750cddd54b54e346ad63